roles/selinux_verify/tasks/main.yml

```diff
@@ -57,12 +60,29 @@
   register: distro_selinux_file_labels
   with_items: "{{ distro_files }}"
 
-- name: Verify distribution specific SELinux file labels are correct
-  fail:
-    msg: "The file {{ item.item.key }} had an incorrect label of {{ item.stdout }} (expected {{ item.item.value }})"
+- name: Append failed distro file labels to the list
+  set_fact:
+    sv_failed_file_labels: "{{ sv_failed_file_labels|default([]) + [ item.item.key + ': ' + item.stdout ] }}"
   when: item.item.value not in item.stdout
   with_items: "{{ distro_selinux_file_labels.results }}"
 
```

Comment:
We lost the expected value which is also also useful in the triage process but I think it can still be added.  Maybe something like:
```
set_fact:
  sv_failed_file_labels: "{{ sv_failed_file_labels|default([]) + [ item.item.key + ': ' + item.stdout + '(expected:' + item.item.value + ')' ] }}"
```

Reply:
Heh.  I was wondering if someone was going to mention that.  Should be easy to add that in.